seed_isort_config.py

```diff
@@ -8,11 +8,16 @@
 import os.path
 import re
 import subprocess
+from collections import OrderedDict
 
 from aspy.refactor_imports.classify import classify_import
 from aspy.refactor_imports.classify import ImportType
 
-
+CONFIG_FILE_TO_START_RE = OrderedDict([
+    ('.isort.cfg', re.compile(r'^(\[settings\])$', re.M)),
```

Comment:
I don't think you need these to be regexes at all -- you're basically just doing verbatim `__contains__`

I also think you can replace this entire mapping with

```python
'[settings]' if filename == '.isort.cfg' else '[isort]'
```

Reply:
The regex makes the file contents substitution possible. Also note `.editorconfig` uses `[*.py]`

Reply:
or `[*]` or `[**.py]` it seems :'(

Reply:
also `.isort.cfg` can also use `[isort]`: https://github.com/timothycrosley/isort/blob/d14d122ab03abcd76e3153270cd8b91854baf889/isort/settings.py#L147-L150

Reply:
🚪 🚶